### PR TITLE
Repo refresh

### PR DESCRIPTION
**Pull request by kbdharun**
This is a massive PR the changes in the PR are as follows

1. Make the title of article cards clickable.
2. Adding spaces between markdown heading as per markdownlint community specification.
3. Adding information on translating the pages along with relevant links.
4. Adding information about markdown and the format for posts along with structures.
5. General rewording of the words in the repository.

NOTE: I will be requesting others opinion as this is a massive PR.

Signed-off-by: K.B.Dharun Krishna <kbdharunkrishna@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,55 @@
 # Vanilla OS Handbook
-This is the official handbook for Vanilla OS. It contains tutorials, guides and other information to help you get the most out of your Vanilla OS installation.
+
+- This is the official handbook for Vanilla OS. It contains tutorials, guides and
+other information to help you get the most out of your Vanilla OS installation.
+- Vanilla OS Handbook utilises Jekyll for generating pages.
+- Thanks for your interest in our project. This guide is for writing and translating articles.
```
Comment (kbdharun) [inline README.md:6]: This can be removed if it isn't necessary. What do you guys think?

File: README.md [hunk 2]
```diff
@@ -1,7 +1,55 @@
...
+description:
+date: YYYY-MM-DD
+layout: article
+authors: 
+    - 1st author GitHub username
+    - 2nd author GitHub username
+    - ...
+---
+
+Contents
```
Comment (kbdharun) [inline README.md:34]:
```suggestion
```
This can be removed too. What do you guys think?

File: README.md [hunk 3]
```diff
@@ -1,7 +1,55 @@
...
+- The filename must be in the format `YYYY-MM-DD-title.md`. Don't translate the layout
+and author names to your native language.
+- Add your GitHub username in the translated
+article along with the original author names. 
+- Once you have ensured everything, open a new pull
+request for the translated guide.
+
+## Discussions 
+
+Discussions regarding the handbook are done in the [official Discord server](https://discord.com/invite/34J8PFsk) in [#docs-writing](https://discord.com/channels/1023243680829681704/1035287786330263703) channel.
```
Comment (kbdharun) [inline README.md:54]: The channel link I attached here redirects to our docs-writing channel.

File: README.md [hunk 4]
```diff
@@ -1,7 +1,55 @@
 # Vanilla OS Handbook
-This is the official handbook for Vanilla OS. It contains tutorials, guides and other information to help you get the most out of your Vanilla OS installation.
+
+- This is the official handbook for Vanilla OS. It contains tutorials, guides and
+other information to help you get the most out of your Vanilla OS installation.
+- Vanilla OS Handbook utilises Jekyll for generating pages.
```
Comment (kra-mo) [inline README.md:5]:
```suggestion
- The handbook uses Jekyll for generating pages.
```

File: README.md (hunk 1 above)
Comment (kra-mo) [inline README.md:6]:
```suggestion
- Thank you for your interest in our project. This guide will help you with writing and translating articles.
```

File: README.md [hunk 5]
```diff
@@ -1,7 +1,55 @@
...
 ## Writing a guide
-If you want to write a guide for the handbook, you can do so by adding the
-new article to the `_posts` folder. The filename must be in the format
-`YYYY-MM-DD-title.md`. Then open a new pull request.
+
+- If you want to write a guide for the handbook, add the
+new article to the `_posts` directory.
+- The filename must be in the format
+`YYYY-MM-DD-title.md`. 
+- Add your GitHub username with the co-author's username in the
```
Comment (kra-mo) [inline README.md:14]:
```suggestion
- Add your GitHub username along with the co-authors' in the
```

File: README.md [hunk 6]
```diff
@@ -1,7 +1,55 @@
...
+author field to have a working link when the post is published.
```
Comment (kra-mo) [inline README.md:15]:
```suggestion
author field to have working links when the post is published. 
```

File: README.md [hunk 7]
```diff
@@ -1,7 +1,55 @@
...
-new article to the `_posts` folder. The filename must be in the format
-`YYYY-MM-DD-title.md`. Then open a new pull request.
+
+- If you want to write a guide for the handbook, add the
+new article to the `_posts` directory.
+- The filename must be in the format
+`YYYY-MM-DD-title.md`. 
+- Add your GitHub username with the co-author's username in the
+author field to have a working link when the post is published. 
+- Set the layout as an
```
Comment (kra-mo) [inline README.md:16]:
```suggestion
- Set the layout as
```

File: README.md [hunk 8]
```diff
@@ -1,7 +1,55 @@
...
+`article`.
+- Headings must follow a proper hierarchy.
+- Refer [markdownlint](https://github.com/DavidAnson/markdownlint) to know the best practices for using markdown.
```
Comment (kra-mo) [inline README.md:19]:
```suggestion
- See [markdownlint](https://github.com/DavidAnson/markdownlint) for markdown best practices.
```

File: README.md [hunk 9]
```diff
@@ -1,7 +1,55 @@
...
+- The guide must be structured in the following way at the beginning:-
```
Comment (kra-mo) [inline README.md:20]:
```suggestion
- The guide must use the following structure at the beginning:
```

File: README.md (hunk 2 above)
Comment (kra-mo) [reply README.md:34]: I don't think it's necessary

File: README.md [hunk 10]
```diff
@@ -1,7 +1,55 @@
...
+authors: 
+    - 1st author GitHub username
+    - 2nd author GitHub username
+    - ...
+---
+
+Contents
+```
+
+- Once you have ensured everything, open a new pull
```
Comment (kra-mo) [inline README.md:37]:
```suggestion
- Once you ensured that everything looks correct, open a new pull
```

File: README.md [hunk 11]
```diff
@@ -1,7 +1,55 @@
...
+request for the guide.
+
+## Translating a guide
+
+- If you want to translate an existing guide to your native language, you can add the translated
+article to the `_posts.ab` directory, where `ab` is your language code. Refer to 
+this [page](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) for language codes.
```
Comment (kra-mo) [inline README.md:44]:
```suggestion
[this](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) page for language codes.
```

File: README.md [hunk 12]
```diff
@@ -1,7 +1,55 @@
...
+- The filename must be in the format `YYYY-MM-DD-title.md`. Don't translate the layout
+and author names to your native language.
+- Add your GitHub username in the translated
+article along with the original author names. 
+- Once you have ensured everything, open a new pull
```
Comment (kra-mo) [inline README.md:49]:
```suggestion
- Once you ensured that everything looks correct, open a new pull
```

File: README.md (hunk 3 above)
Comment (kra-mo) [inline README.md:54]:
```suggestion
Discussions regarding the handbook are done in the [official Discord server](https://discord.com/invite/34J8PFsk) in [#docs-writing](https://discord.com/channels/1023243680829681704/1035287786330263703).
```

**Review by kra-mo — CHANGES_REQUESTED** [review]

**Comment by mirkobrombin** [comment]
Nice work!
What about moving the instructions on how to post from README to the handbook directly?

**Comment by kbdharun** [comment]
Sure, I will create a post we can keep this in the readme too as it will be of immediate reference.

**Comment by kbdharun** [comment]
Done, @mirkobrombin. Checked everything with Jekyll locally too.

File: _posts/2022-11-11-contributing.md [hunk 13]
```diff
@@ -0,0 +1,61 @@
+---
+title: Contributing to Handbook
```
Comment (kbdharun) [inline _posts/2022-11-11-contributing.md:2]:
```suggestion
title: Contribute to Handbook
```

File: _posts/2022-11-11-contributing.md [hunk 14]
```diff
@@ -0,0 +1,61 @@
...
+description: Learn how to contribute to Vanilla OS Handbook.
+date: 2022-11-11
+layout: article
+authors: 
+    - kbdharun
+    - kra-mo
+---
+
+# Contributing to Handbook
```
Comment (kbdharun) [inline _posts/2022-11-11-contributing.md:11]:
```suggestion
```

**Comment by kbdharun** [comment]
I will add links to the directories too here.

**Comment by mirkobrombin** [comment]
Nice!

**Pull request merged**